LEGO1/omni/src/common/mxstring.cpp

```diff
@@ -204,5 +204,30 @@ void MxString::NormalizePath(char* p_path)
 
 		path++;
 	}
+
+	size_t pathLen = SDL_strlen(p_path);
+
+	auto mapPath = [p_path, pathLen](const vector<MxString>& p_files) -> bool {
+		for (const MxString& file : p_files) {
+			// Test whether file is a suffix of the provided path (case insensitive)
+			// If yes, copy the original file system path into p_path.
+			for (size_t i = pathLen, j = file.GetLength(); i != 0 && j != 0; i--, j--) {
+				if (SDL_tolower(p_path[i - 1]) != SDL_tolower(file.GetData()[j - 1])) {
+					break;
+				}
+				else if (j == 1) {
+					SDL_strlcpy(&p_path[i - 1], file.GetData(), file.GetLength() + 1);
+					SDL_Log("Resolved file path to %s", p_path);
```

Comment:
I think might become chatty. Perhaps do it as
```suggestion
					SDL_LogTrace("Resolved file path to %s", p_path);
```
The logging level can be change by setting the `SDL_LOGGING` environment variable.
(`SDL3/SDL_hints.h` documents what value the environment variable expects)